docs/intro/glossary.md

```diff
@@ -0,0 +1,241 @@
+
+#### Relay
+
+A device that generally have two states: open (connected) and closed (disconnected).
+
+They are generally placed in front of another [component](#component), e.g., an [inverter](#inverter), to control whether the component is connected to the [microgrid](#microgrid) or not.
+
+#### Sensor
+
+A device for [measuring](#measurement] ambient [metrics](#metric) (for example temperature, humidity, etc.).
+
+#### Solar Panel
+
+A panel with [PV](#pv) cells that generates [DC](#dc) electricity from sunlight.
+
+#### Wind Turbine
+
+A device that converts the wind's kinetic energy into electrical energy.
+
+### Component
+
+A device (of a particular [category](#component-category)) within a [microgrid](#microgrid).
+
+### Component ID
+
+A numeric identifier uniquely representing an instance of a [component](#component) in a particular[microgrid](#microgrid). It is always of type `int`.
+
+For example, a battery an have a component ID **5**.
+
+### Island
+
+A [microgrid](#microgrid) that is not connected to the public electricity [grid](#grid).
+
+## Component Data
+
+### Metric
+
+A quantifiable attribute of a [component](#component).
+
+For example, the metric **capacity** of a battery.
+
+### Measurement
+
+An individual numeric value obtained from a [metric](#metric) of a [component](#component) instance. It is consistently of type `float`, but it is often expressed in specific units.
+
+In the context of a sample, this is commonly referred to as a *sample value*.
+
+For example, a measurement of the capacity of a battery with component ID 5 can be **400**, typically measured in Watt-hours (Wh).
+
+### Timestamp
+
+A specific point in time, always represented as a `datetime` with a `timezone` attached.
+
+For example, **2022-01-01 22:00:00.000 UTC**.
+
+### Sample
+
+A [measurement](#measurement) recorded at a particular [timestamp](#timestamp), typically represented as a tuple `(timestamp, value)`.
+
+For example, recording a measurement of 400 from the capacity of a battery at 2022-01-01 22:00:00.000 UTC would constitute a sample **`(2022-01-01 22:00:00.000 UTC, 400)`**.
+
+#### Sample Value
+
+A [measurement](#measurement) stored in a [sample](#sample).
+
+### Time Series
+
+A sequence of [samples](#sample), often organized by [timestamp](#timestamp) and typically with regular intervals. However, irregular (event-based) time series are also possible.
+
+For example, a time series representing measurements of a battery's capacity at 2022-01-01 22:00:00.000 UTC every second for 5 seconds would appear as follows:
+
+```
+(2022-01-01 22:00:00.000 UTC, 400)
+(2022-01-01 22:00:01.000 UTC, 401)
+(2022-01-01 22:00:02.000 UTC, 403)
+(2022-01-01 22:00:03.000 UTC, 402)
+(2022-01-01 22:00:04.000 UTC, 403)
+```
+
+### Timeseries
+
+Same as [time series](#time-series).
```

Comment:
obvious.

Reply:
Is for searching. If you search for "time series" in the search bar, "timeseries" won't be shown as a result.